task.dart

```diff
@@ -0,0 +1,118 @@
+import 'dart:io';
+
+void main() {
+  bool flag = true;
+  var myList = [];
+  print("How many tasks do you want to do : \n");
+  int numberOfTask = int.parse(stdin.readLineSync()!);
+
+  print("Enter your tasks to do: \n ");
+  for (int i = 0; i < numberOfTask; i++) {
+    var taskToDo = stdin.readLineSync();
+    myList.add(taskToDo);
+  }

```

Comment:
Lists in dart are dymanic and dont need a specific size. Also, you should create a model for your tasks. Here is an example of a model
```dart
  class TaskModel
  {
    String title;
    String description;
    int id;

    TaskModel({
        this.title = "",
        this.description = "",
        this.id = 0,
    })
  }
```